PeakLocalisations/Localisation/App Store/InfoPlist.xcstrings

```diff
@@ -80,6 +104,12 @@
             "state" : "new",
             "value" : "Peak saves your reports to your photo library"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Peak guarda sus informes en su biblioteca de fotos"
```

Comment:
this might be libreria in spanish too? instead of biblioteca?

Reply:
I was confused between the two but Apple seems to use biblioteca: https://applelocalization.com/?q=library&l=English&l=Spanish

Reply:
Prefer biblioteca. Librería means book store / bookshop